testjs.js

```diff
@@ -0,0 +1,44 @@
+// 1. Programing Basics 
+
+const dishes = [
+    'Spaghetti',
+    'Lasagne',
+    'Beans on toast',
+    'Porridge',
+    'Tomato soup',
+    'Salmon'
+];
+
+for (let i = 0; i < dishes.length; i++) {
+    console.log(dishes[i]);
+}
+
+
+//2. Dom manipulation 
+
+const btn = document.createElement('button');
+btn.innerText = 'Click me';
+document.body.appendChild(btn);
+
+btn.addEventListener('click', (e) => {
+    const header = document.createElement('h1')
+    header.innerHTML = 'Js Test';
+    document.body.appendChild(header);
+    document.body.style.backgroundColor = `#${Math.floor(Math.random() * 16777215).toString(18)}`
+});
+
```

Comment:
Very nice and some extra touches 🤓 

The console log text is not what is asked in the task  👀 

A possible reconsideration would be to not create and populate the inner HTML of a new header on every click.
You could have it ready outside of the event listener callback and just use it when needed, as it is an element that does not change and therefore there is no need to load that part of the process onto the event callback.